docs_outdoornav_user_manual/features/mission_scheduler.mdx

```diff
@@ -5,3 +5,40 @@ sidebar_position: 4
 toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
+
+### Scheduling missions
+
+By selecting the "SCHEDULER" option from the upper left hand hamburger menu, the user can open the Mission scheduler interface. This interface 

```

Comment:
1. Do we call it the "hamburger" menu elsewhere? I suspect we have a different name for it. Just want to ensure we are consistent.
2. "Mission scheduler" -> "Mission Scheduler"

Reply:
1. I did a quick search to see what we call it and there are a lot of areas where it is called a hamburger menu though there are other places that it's called just a menu. I can't say for certain but it seems like mostly when it's called hamburger menu it's from sections that I worked on so I may have introduced this. I can remove it and replace them all with just "menu" if that's preferred.

Reply:
I'm ok either way, as long as we are consistent